cc/configure_tensorflow.sh

```diff
@@ -60,29 +58,54 @@ echo "Copying tensor flow headers to ${dst_dir}"
 cp -r ${tmp_dir}/tensorflow-*.data/purelib/tensorflow/include/* "${dst_dir}"
 
 echo "Building tensorflow libraries"
-bazel build -c opt --config=opt --copt="${cc_opt_flags}" //tensorflow:libtensorflow_cc.so //tensorflow:libtensorflow_framework.so
+
+# Add a custom BUILD target for the gRPC runtime.
```

Comment:
this is both hilarious and awful.

Reply:
and i'm also impressed -- i don't think i would've come up with this as a solution XD